src/index.tsx

```diff
@@ -7,12 +7,12 @@ import { IsMobileProvider } from "./is-mobile";
 import { App } from "./components/App";
 import "./styles.scss";
 
-const SentyEnvHostnameMap: { [key: string]: string } = {
+const SentryEnvHostnameMap: { [key: string]: string } = {
```

Comment:
I feel sort of silly for including these "typo" fixes in the PR. I just couldn't bear my spellcheck yelling at me 😄